### PR TITLE
Refactor/req to fetch

### PR DESCRIPTION
**Pull request by dogpatch626**
step away from request as the library is no longer being maintained.
## How Has This Been Tested?

tested with unit tests and packing/installing into one app itself running one app tests.

## Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation (adding or updating documentation)
- [x] Dependency update

## Checklist:

- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] My changes are in sync with the code style of this project.
- [ ] There aren't any other open Pull Requests for the same issue/update.
- [ ] These changes should be applied to a maintenance branch.
- [ ] I have added the Apache 2.0 license header to any new files created.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -7746,7 +7746,7 @@ replace-ext@1.0.0:
   resolved "https://registry.yarnpkg.com/replace-ext/-/replace-ext-1.0.0.tgz#de63128373fcbf7c3ccfa4de5a480c45a67958eb"
   integrity sha1-3mMSg3P8v3w8z6TeWkgMRaZ5WOs=
 
-request@^2.79.0, request@^2.88.0, request@^2.88.2:
+request@^2.79.0, request@^2.88.0:
```
Comment (Matthew-Mallimo) [inline yarn.lock:7749]: Did you manually change the `package.json` dependencies? If you did `yarn remove request` I would think this wouldn't be here

Comment (dogpatch626) [reply yarn.lock:7749]: I did yarn remove

Comment (Matthew-Mallimo) [reply yarn.lock:7749]: wonder why thats left behind then

Comment (dogpatch626) [reply yarn.lock:7749]: I have no clue, I did yarn remove again and seeing that its not in the package.json it just exited

**Review by giulianok — APPROVED** [review]

**Review by 10xLaCroixDrinker — APPROVED** [review]

**Review by Matthew-Mallimo — APPROVED** [review]

**Pull request merged**